src/puppetlabs/jdbc_util/core.clj

```diff
@@ -156,3 +156,31 @@
                        [sql 0]
                        seq-params-w-indices)]
     (vec (conj (flatten parameters) sql'))))
+
+(defn has-extension [db extension]
+  (-> (jdbc/query db ["select count(*) from pg_extension where extname = ?" extension])
+      first
+      :count
+      pos?))
+
+(defn has-pglogical-extension [db]
+  (has-extension db "pglogical"))
+
+(defn- unsafe-escape-sql-string
```

Comment:
Since the only thing this does is escape single quotes, do you think it's reasonable to name it to reflect that? Even though it has a pretty risque name I don't want anybody to think it does more than it does.